rllib/env/vector_env.py

```diff
@@ -274,44 +275,70 @@ class VectorEnvWrapper(BaseEnv):
     def __init__(self, vector_env: VectorEnv):
         self.vector_env = vector_env
         self.num_envs = vector_env.num_envs
+
+        # This VectorEnv is multi-agent.
+        self.is_multiagent = False
+        for sub_env in self.vector_env.get_sub_environments():
```

Comment:
should this check live in Avnish's VectorEnv validation module?
we would simply do the following here.
```
self.is_multiagent = any(
    isinstance(e, MultiAgentEnv) for e in self.vector_env.get_sub_environments())
```